ui/page/overview/sync_details.go

```diff
@@ -64,6 +64,12 @@ func (pg *AppOverviewPage) syncStatusTextRow(gtx layout.Context, inset layout.In
 		return layout.Flex{Axis: layout.Horizontal, Spacing: layout.SpaceBetween}.Layout(gtx,
 			layout.Flexed(1, syncStatusLabel.Layout),
 			layout.Rigid(func(gtx C) D {
+
+				// Set gxt to Disabled (Sets Queue to nil) if syncClickable state is disabled, prevents double click.
+				if !pg.syncClickable.Enabled {
+					gtx = gtx.Disabled()
+				}
```

Comment:
Why not handle the disabled functionality from the clickable widget? `pg.syncClicable.SetEnable(enabled bool)`

Reply:
Nice idea, I had thought about that but realised it require heavy modification of the widget.